### ISSUE TITLE
Responsive design (CSS)

### ISSUE THREAD

**Issue by Engineervinay** [Engineervinay/threads-clone#3]
The current website is not responsive and needs to be zoomed out a little to view, which can be changed the mobile view is not good that can be improved.
Edit : Note: Create separate issue if you have to improve particular components.
currently we are not focusing on desktop view so the UI improvement will be for mobile but we can also finalize some structure for Desktop by discussing.

**Comment by Om2123** [comment]
use tailwind css

**Comment by mank-423** [comment]
I can try using Tailwind CSS to make it more responsive and the CSS files more modular, if you want these changes in your project @Engineervinay

**Comment by Om2123** [comment]
> I can try using Tailwind CSS to make it more responsive and the CSS files more modular, if you want these changes in your project

why would you , this is not any open source project . just a personal project

**Comment by Engineervinay** [comment]
@mank-423 okay, but can you please try it using vanilla CSS we are doing this project to improve and brush up our skills so it will be better if we use CSS instead of Tailwind. and now shifting all the CSS into tailwind will be a hectic task.
if you are okay with CSS then you can create separate issues for different components.

**Comment by mank-423** [comment]
Ok @Engineervinay I will try to work with CSS only to make it responsive, just tell me one thing, we just want to make it responsive for the mobile view right?

**Comment by Engineervinay** [comment]
Yes, currently we have zoomout of the application to use it even on desktop if we solve that issue it will also solve the responsiveness, for some mobiles its working but for some screen sizes its not working.

**Comment by Abhineshhh** [comment]
@mank-423 if you are not doing this issue can i give it a try

**Comment by mank-423** [comment]
@AbhineshJha hey hi I am currently trying to work on it, converting tailwind knowledge to pure css, and yess why not you should also try for it

**Comment by Engineervinay** [comment]
Assigned this issue to both @mank-423 @AbhineshJha

**Comment by HannesAlexandersson** [comment]
Is this issue still open?
